srv/wordpress/wp-content/plugins/acore-wp-plugin/src/Components/AdminPanel/Pages/SoapLogs.php

```diff
@@ -0,0 +1,146 @@
+                                                echo "<li class=\"page-item disabled\"><a class=\"page-link\">...</a></li>";
+                                            }
+                                            echo "<li class=\"page-item\"><a class=\"page-link\" href=\"$link&pos={$data["max_page"]}\">{$data["max_page"]}</a></li>";
+                                        }
+                                    ?>
+                                </ul>
+                            </nav>
+                        </div>
+
+                    </form>
+                    <table class="table table-bordered table-hover align-middle">
+                        <thead>
+                            <tr>
+                                <th>#</th>
+                                <th>Result</th>
+                                <th>Details</th>
+                                <th>Command</th>
+                                <th>User</th>
+                                <th>Order</th>
+                                <th>Executed DateTime</th>
+                            </tr>
+                        </thead>
+                        <tbody>
+                            <?php
+                            foreach ($result as $item) {
+                                echo "<tr><td>{$item->id}</td>";
+                                if ($item->success) {
+                                    echo "<td><span class=\"dashicons dashicons-yes text-success\"></span></td>";
+                                } else {
+                                    echo "<td><span class=\"dashicons dashicons-no text-danger\"></span></td>";
+                                }
+                                echo "<td>{$item->result}</td>";
+                                echo "<td>{$item->command}</td>";
+                                $user_info = get_userdata($item->user_id);
+                                echo "<td><a href=\"" . get_edit_user_link($item->user_id) . "\">{$user_info->user_login}</a></td>";
+                                echo "<td>{$item->order_id}</td>";
+                                echo "<td>{$item->executed_at}</td></tr>";
+                            }
+                            ?>
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        </div>
+    </div>
+    <hr />
+    <script>
+        jQuery('#preview').on('click', function (e) {
+            jQuery('#pvp-rewards').attr('method', 'GET');
+        });
+        jQuery('#pvp-rewards').on('submit', function (e) {
+            return confirm("You sure you want to continue?");
+        });
+        jQuery(document).on('ready', function () {
+            var tooltipTriggerList = [].slice.call(document.querySelectorAll('[data-bs-toggle="tooltip"]'));
+            tooltipTriggerList.forEach(function (tooltipTriggerEl) {
+                return new bootstrap.Tooltip(tooltipTriggerEl)
+            });
```

Comment:
for-of should be better of forEach in terms of performance 
```suggestion
            for (const tooltipTriggerEl of tooltipTriggerList) {
                new bootstrap.Tooltip(tooltipTriggerEl);
            };
```